bin/tx-prover/Cargo.toml

```diff
@@ -49,7 +49,9 @@ miden-objects = { workspace = true, default-features = false }
 miden-tx = { workspace = true, default-features = false }
 prost = { version = "0.13", default-features = false, features = ["derive"] }
 rand = "0.8"
+reqwest = { version = "0.11" }
 serde = { version = "1.0", features = ["derive"] }
+serde_qs = { version = "0.13" }
```

Comment:
Not for this PR, but we should separate the dependencies which are needed for the binary vs. the ones that are needed for the library. One way to do this is by introducing an `executable` feature, but there could be other ways as well. Let's create an issue for this.

Reply:
I create the issue #1001 to track this.